src/templates/components/fonts/Egyptian.css

```diff
@@ -1,6 +1,7 @@
 @font-face {
 	font-family: 'GuardianTextEgyptian';
-	src: url('https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-Regular.woff2')
+	src:
+		url('https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-Regular.woff2')
 			format('woff2'),
 		url('https://assets.guim.co.uk/static/frontend/fonts/guardian-textegyptian/noalts-not-hinted/GuardianTextEgyptian-Regular.woff')
```

Comment:
I can't tell what the changes are in these files?

Reply:
the linter has added newlines, now using guardian prettier settings